docroot/modules/custom/app_module/tests/src/Kernel/AppPathTest.php

```diff
@@ -141,9 +142,10 @@ private function checkNodeWithData() {
     // Reload the node to make sure that the data survived.
     $this->nodeStorage->resetCache();
     $node1 = Node::load($node1->id());
+    /* @phpstan-ignore-next-line */
     $this->assertEquals($node1->field_application_module->data, $data);
-
     // 2. Change data.
+    /* @phpstan-ignore-next-line */
     $node1->field_application_module->data = $data2;
```

Comment:
Can not resolved due to Access to an undefined property data because we are updating the field value and the syntax for storing any attribute is like this only.